### ISSUE TITLE
Ensure full accessibility

### ISSUE THREAD

**Issue by wwalc** [ckfinder/ckfinder#91]
CKFinder should be compliant with: WCAG AA (same as CKEditor) + Section 508.

Proper WAI-ARIA attributes should be used.

**Comment by wwalc** [comment]
Closing as too generic, let's have separate tickets for each issue that still have left. Significant accessibility improvements were introduced in version 3.1.
